tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml

```diff
@@ -0,0 +1,12 @@
+host = "localhost"
+port = 3306
+user = "root"
+database = ""
+password = ""
+
+[foo]
+host = "wso2.com"
+port = 6688
+user = "john"
+password = "john123"
+database = "mariadb"
```

Comment:
```suggestion
[foo]
host = "wso2.com"
port = 6688
user = "john"
password = "john123"
database = "mariadb"

host = "localhost"
port = 3306
user = "root"
database = ""
password = ""
```

Reply:
Let's update the expected output as above and disable this test case for now. Re-enable after the issue is sorted.

Reply:
sorted in [3cb66d3](https://github.com/ballerina-platform/persist-tools/pull/7/commits/3cb66d3d9ba065943e9b6d1b8df81da6d6f56135)